tests/migration_test.go

```diff
@@ -640,15 +622,11 @@ var _ = Describe("[Serial][rfe_id:393][crit:high][vendor:cnv-qe@redhat.com][leve
 					log.DefaultLogger().Infof("expoected time: %v", expectedTime)
 
 					By("Checking that the guest has an updated time")
-					resp, err := console.ExpectBatchWithValidatedSend(expecterNew, []expect.Batcher{
+					err := console.SafeExpectBatch(vmi, []expect.Batcher{
 						&expect.BSnd{S: "date +%H:%M\n"},
 						&expect.BExp{R: expectedTime},
-					}, 30*time.Second)
-					if err != nil {
-						log.DefaultLogger().Infof("time in the guest %v", resp)
-						return false
-					}
-					return true
+					}, 30)
+					return err == nil
```

Comment:
Can we change the return type to `error` and do `ShouldNot(HaveOccurred())` in case of failure ginkgo will print the error.